Bottom: 5c07a582f64d8374639ef2e5ff5c59573175cd8a
Top:    2c8adaebad1605059001930b1ae3ce89c1f14a5d
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-29 20:01:23 +0100

hid/gtk: Don't refleak the accel group created by the layer selector

When the accel group is created, we have a reference to it - we don't need
another.


---

diff --git a/src/hid/gtk/ghid-layer-selector.c b/src/hid/gtk/ghid-layer-selector.c
index 639e2be..840cd98 100644
--- a/src/hid/gtk/ghid-layer-selector.c
+++ b/src/hid/gtk/ghid-layer-selector.c
@@ -431,8 +431,6 @@ ghid_layer_selector_new (void)
     g_signal_connect (ls->selection, "changed",
                       G_CALLBACK (selection_changed_cb), ls);
 
-  g_object_ref (ls->accel_group);
-
   return GTK_WIDGET (ls);
 }
